packages/base-styles/_mixins.scss

```
@@ -603,4 +670,4 @@
 	bottom: 0;
 	left: 0;
 	@include selected-block-outline($widthRatio);
-}
+}
```

No need for removing the last line

```suggestion
}

```